### PR TITLE
Update Overview.md

### PR DESCRIPTION
**Pull request by klues**

### CODE REVIEW

File: Documentation/docs/get-started/Overview.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ This pages explains the most important terms and elements of the AsTeRICS framew
 
 ## Solutions
 
-The AsTeRICS framework provides some selected [solutions](/solutions/) which can be used out of the box. The solutions are grouped by **use cases** and can be directly started from the solutions page. Some solutions depend on certain [input devices](#input-device) or other hardware and software. Just click onto the ```Read More``` button in a solution's tile to get a detailed step by step instruction of how to install and use it.
+The AsTeRICS framework provides some selected [solutions](/solutions/) which can be used out of the box. The solutions are grouped by **use cases** and can be directly started from the solutions page. Some solutions depend on certain [input devices](#input-device) or other hardware and software. Just click on the ```Read More``` button in a solution's tile to get a detailed step by step instruction of how to install and use it.
```
Comment (klues) [inline Documentation/docs/get-started/Overview.md]: click "on" should be correct, see https://dict.leo.org/forum/viewGeneraldiscussion.php?idforum=4&idThread=1182156&lp=ende&lang=de

File: Documentation/docs/get-started/Overview.md [hunk 2]
```diff
@@ -4,14 +4,10 @@ This pages explains the most important terms and elements of the AsTeRICS framew
...
 
 A solution can be categorized by the AsTeRICS technology used. It can be a [model](#model), a [grid](#grid) or an [AT solution](#at-solution).
 
-## Terms
```
Comment (klues) [inline Documentation/docs/get-started/Overview.md]: why an own section `terms` after there was already explained the first term `Solution`?

File: Documentation/docs/get-started/Overview.md [hunk 3]
```diff
@@ -4,14 +4,10 @@ This pages explains the most important terms and elements of the AsTeRICS framew
...
-
-The most important terms are described below.
-
 ### Model
 
 A model represents a simple Assistive Technology (AT). It is used to define **what** should be done (e.g. mouse control, switching light on/off) and **how** this can be done (e.g. button-press, head movement,...). A model uses [plugins](#plugin) to support input devices (e.g. webcam, switch,...) and required actions.
```
Comment (klues) [inline Documentation/docs/get-started/Overview.md]: maybe one sentece to describe the image?

**Comment by klues** [comment]
Actions -> Plugins/Actuators is also a dead link

File: Documentation/docs/get-started/Overview.md [hunk 4]
```diff
@@ -33,7 +29,7 @@ The actions that are executed by a model are implemented using actuator plugins,
 
 ### Grid
 
-The grid is a web-based user interface which can be used for Augmentative and Alternative Communication (AAC). It is organized as grid of rows and columns with cells acting as buttons. The cells support texts and images/symbols and can speak a defined text (text-to-speech) or execute a defined action. A grid can be connected to a model and be used to trigger actions like switching lights on/off.
+The AsTeRICS Grid is a web-based user interface which can be used for Augmentative and Alternative Communication (AAC). Grids can be defined in a flexible layout of cells acting as buttons. The cells show text and images/symbols and if one of them is selected a defined phrase can be spoken (text-to-speech). A grid can also be connected to a model and be used to trigger actions like switching lights on/off.
 
 ![Grid with symbols for Alternative and Augmentative Communication](/img/AsTeRICS-Ergo_Grid_en-1-768x592.jpg)
```
Comment (klues) [inline Documentation/docs/get-started/Overview.md]: this image is not shown

**Comment by klues** [comment]
ACS user manual -> dead link

**Comment by klues** [comment]
> Additionally, a model can be uploaded, if opened in the ACS/WebACS programs or directly from the solutions page.

I do not understand this sentence.

File: Documentation/docs/get-started/Overview.md [hunk 5]
```diff
@@ -61,7 +57,7 @@ The WebACS is a new web-based version of the [ACS](#acs) program for designing a
...
+The AsTeRICS Runtime Environment (ARE) executes a given model. The screenshot below shows the autostart model running after starting the ARE. You can click the ```Play```, ```Pause``` or ```Stop``` button of the control panel to start, pause or stop a model. Alternatively, you can use the respective shortcuts ```F5```,```F6```,```F7```. Furthermore, you can open a new model file by clicking onto the ```Open``` button. Additionally, a model can be uploaded, if opened in the ACS/WebACS programs or directly from the [solutions](/solutions/) page. The ARE also provides a remote interface. For more information about how to use the ARE, please read the [ARE user manual](/manuals/ARE/).
```
Comment (klues) [inline Documentation/docs/get-started/Overview.md]: > The ARE also provides a remote interface

Which remote interface?

Comment (klues) [reply Documentation/docs/get-started/Overview.md]: `/manuals/ARE/` is a dead link

File: Documentation/docs/get-started/Overview.md (hunk 1 above)
Comment (deinhofer) [reply Documentation/docs/get-started/Overview.md]: ok

File: Documentation/docs/get-started/Overview.md (hunk 2 above)
Comment (deinhofer) [reply Documentation/docs/get-started/Overview.md]: good question, I thought just because to mention the solutions first.

**Comment by deinhofer** [comment]
ok, thanks, will merge

**Pull request merged**